test/tools/ossfuzz/protoToYul.h

```diff
@@ -127,6 +134,12 @@ class ProtoConverter
 				break;
 		}
 	}
+	/// Returns a pseudo-random dictionary token.
+	std::string getDictionaryToken(Prefix _p = Prefix::ADD);
```

Comment:
```suggestion
	std::string dictionaryToken(Prefix _p = Prefix::ADD);
```
`get` is discouraged in the coding style